src/sql/workbench/parts/query/browser/queryResultsView.ts

```diff
@@ -103,6 +103,40 @@ class ResultsView extends Disposable implements IPanelView {
 	}
 }
 
+class SandDanceView extends Disposable implements IPanelView {
+	private container = document.createElement('div');
```

Comment:
So right now this is just adding the button but it doesn't appear to actually display anything. What's the plan for implementing that behavior? You probably don't want to actually add the action until the tab will do something either